addons/web/static/src/core/file_input/file_input.js

```diff
@@ -54,6 +56,25 @@ export class FileInput extends Component {
     }
 
     async uploadFiles(params) {
+        if (params.ufile.length) {
+            const fileSize = params.ufile[0].size;
+            if (fileSize > this.maxUploadSize) {
+                this.notification.add(
+                    this.env._t(
+                        `The selected file (${(fileSize / (1024 * 1024)).toFixed(
+                            2
+                        )} MB) is over the maximum allowed file size (${(
+                            this.maxUploadSize /
+                            (1024 * 1024)
+                        ).toFixed(2)} MB).`
+                    ),
```

Comment:
C'est pas très lisible dans l'état actuel :grimacing: 
```suggestion
                    const fileSizeMB = (fileSize / (1024 * 1024)).toFixed(2);
                    const maxUploadSizeMB = (this.maxUploadSize / (1024 * 1024)).toFixed(2);
                    this.env._t(
                        `The selected file (${fileSizeMB} MB) is over the maximum allowed file size (${maxUploadSizeMB} MB).`
                    ),
```

Reply:
IIRC interpolated strings does not work with the translation mechanism.
Instead you should import and use `sprintf` alongside the `_t` function.